hack/verify-generated-protobuf.sh

```diff
@@ -16,6 +16,8 @@ for APIROOT in ${APIROOTS}; do
   cp -rf "${OS_ROOT}/${APIROOT}" "${_tmp}/"
 done
 
+go get -u golang.org/x/tools/cmd/goimports
```

Comment:
This seems suspect but should be fine

Reply:
This file is removed in master, so I used a part of `vendor/github.com/coreos/etcd/scripts/genproto.sh` to fix this